modules/src/core/ics04_channel/handler/chan_open_try.rs

```diff
@@ -125,14 +136,18 @@ pub(crate) fn process(
     // Transition the channel end to the new state & pick a version.
     new_channel_end.set_state(State::TryOpen);
 
+    let (channel_id_state, channel_cap) = if msg.previous_channel_id.is_none() {
```

Comment:
We only create a new capability only if a `previous_channel_id` isn't specified.